dataland-frontend/tests/e2e/specs/company-metadata/CompaniesOnlySearch.ts

```diff
@@ -1,18 +1,20 @@
+import { describeIf } from "@e2e/support/TestUtility";
+import { CompanyInformation, EuTaxonomyDataForFinancials } from "@clients/backend";
+
 describe("As a user, I want to be able to search companies existing on Dataland", function () {
   beforeEach(() => {
     cy.ensureLoggedIn();
   });
 
-  it("Check if the search bar is available, and if the show-all-companies button works as expected", function () {
-    function verifyCompanySearchResultTable(): void {
-      cy.get("table.p-datatable-table").contains("th", "COMPANY");
-      cy.get("table.p-datatable-table").contains("th", "SECTOR");
-      cy.get("table.p-datatable-table").contains("th", "MARKET CAP");
-    }
+  function verifyCompanySearchResultTable(): void {
+    cy.get("table.p-datatable-table").contains("th", "COMPANY");
+    cy.get("table.p-datatable-table").contains("th", "SECTOR");
+  }
+  const inputValue = "d-fine";
```

Comment:
Warum hast du das `inputValue` aus dem it-scope herausgehoben. Das wird doch nur in dem scope verwendet oder nicht?

Reply:
das war noch ein Zwischenstand. Der Test ist jetzt noch weiter aufgeräumt

Reply:
Ok